pkg/pillar/zedcloud/tls.go

```diff
@@ -296,3 +304,49 @@ func stapledCheck(connState *tls.ConnectionState) bool {
 	}
 	return resp.Status == ocsp.Good
 }
+
+func updateEtcSSLforProxyCerts(ctx *ZedCloudContext, dns *types.DeviceNetworkStatus) {
+	if !ctx.V2API || ctx.AgentName != "zedagent" {
+		log.Infof("updateEtcSSLforProxyCerts: skip agent %s\n", ctx.AgentName)
+		return
```

Comment:
So all of the other agents (logmanager, downloader) will fight over modifying the same shared directory? That seems unnecessary and risky.

Reply:
this check is saying only 'zedagent' will update the shared directory. other agents just return.

Reply:
Duh - I can't read.